### ISSUE TITLE
Print in mobile (tablet)

### ISSUE THREAD

**Issue by zeltos** [at-point-of-sale/ReceiptPrinterEncoder#7]
using laptop successfully completed print as expected, but when trying print via mobile (tablet/or smartphone) printer not fully print all page. 

anyone have this issue?

**Comment by bauermar** [comment]
Yes same here, from mobile phone only some characters will be printed. Maybe a buffering problem?

**Comment by NielsLeenheer** [comment]
Sounds like a buffering issue with the communication issue between your app and the printer. Could indeed be buffering. The app is sending the data quicker than the printer can print it. That means the printer must store the data in its memory. If that memory is not large enough to store the whole page, it will ignore the rest once the buffer is full. 

Not really an issue with this library as it just generates the data. The issue is probably with the library you are using to send the data. Out of curiosity, what library are you using and which transport method to printer (usb, serial, bluetooth, network)?

**Comment by NielsLeenheer** [comment]
Closing this due to inactivity. The symptoms point at the app sending data faster than the printer can buffer it, which sits in the transport library rather than in the encoder. If it is still a problem, open a new issue with the library and connection type you use to send the data, and we can take it from there.
